test/common/http/conn_manager_impl_test_base.cc

```diff
@@ -40,19 +40,19 @@ HttpConnectionManagerImplTest::HttpConnectionManagerImplTest()
   ip_detection_extensions_.push_back(getXFFExtension(0));
 }
 
-HttpConnectionManagerImplTest::~HttpConnectionManagerImplTest() {
+HttpConnectionManagerImplMixin::~HttpConnectionManagerImplMixin() {
   filter_callbacks_.connection_.dispatcher_.clearDeferredDeleteList();
 }
 
 Tracing::CustomTagConstSharedPtr
-HttpConnectionManagerImplTest::requestHeaderCustomTag(const std::string& header) {
+HttpConnectionManagerImplMixin::requestHeaderCustomTag(const std::string& header) {
   envoy::type::tracing::v3::CustomTag::Header headerTag;
   headerTag.set_name(header);
   return std::make_shared<Tracing::RequestHeaderCustomTag>(header, headerTag);
 }
 
-void HttpConnectionManagerImplTest::setup(bool ssl, const std::string& server_name, bool tracing,
-                                          bool use_srds) {
+void HttpConnectionManagerImplMixin::setup(bool ssl, const std::string& server_name, bool tracing,
+                                           bool use_srds) {
```

Comment:
nit: space before the param type? Here and a few places below

Reply:
hmm, I thought the fix_format command will take care of this.

Reply:
wait, I think we want this space so that the bool aligns with the beginning of the () content, i.e. the first "bool".